tests/functional_tests/strategy_evaluators_tests/abstract_strategy_test.py

```diff
@@ -42,62 +44,47 @@ class AbstractStrategyTest(AbstractBacktestingTest, ABC):
     __metaclass__ = ABCMeta
 
     def initialize(self, strategy_evaluator_class, trading_mode_class=DailyTradingMode, config=None):
-        if config is None:
-            self.config = create_backtesting_config(load_test_config(), filter_symbols=False)
-        else:
-            self.config = config
+        self.logger = get_logger(self.__class__.__name__)
+        self.config = load_test_config() if config is None else config
         self.strategy_evaluator_class = strategy_evaluator_class
         self.trading_mode_class = trading_mode_class
-        self._register_evaluators()
-        self._register_only_strategy(strategy_evaluator_class)
-        self._register_only_trading_mode(trading_mode_class)
-        self._assert_init()
-
-    def _assert_results(self, run_results, profitability, bot):
-        # convenient for building tests
-        # print(f"results: {run_results} expected: {profitability} => {run_results[0] >= profitability}")
+        self.tentacles_setup_config = asyncio.run(load_test_tentacles_config())
+        self._update_tentacles_config(strategy_evaluator_class, trading_mode_class)
 
-        assert run_results[0] >= profitability
+    def _handle_results(self, independent_backtesting, profitability):
+        exchange_manager_ids = get_independent_backtesting_exchange_manager_ids(independent_backtesting)
+        for exchange_manager in get_exchange_managers_from_exchange_ids(exchange_manager_ids):
+            _, run_profitability, _, market_average_profitability, _ = get_profitability_stats(exchange_manager)
+            # convenient for building tests
+            print(f"results: run profitability {run_profitability} market profitability: {market_average_profitability}"
+                  f" expected: {profitability} => {run_profitability >= profitability}")
+            assert run_profitability >= profitability
```

Comment:
We accept superior profitability ?

Reply:
that's how it was previously, actually it might be bettter with a 
```python
profitability * 1.1 =>  run_profitability >= profitability
```
What do you think ?

Reply:
In fact I don't understand why it's not `assert run_profitability == profitability` ?

Reply:
Not sure why it was like this in 0.3, let's put a == for now and be optimistic :)